src/Sylius/Bundle/PayumBundle/composer.json

```diff
@@ -31,12 +31,12 @@
         "php": "^8.0",
         "payum/payum": "^1.7.2",
         "payum/payum-bundle": "^2.5",
-        "php-http/guzzle6-adapter": "^2.0",
         "sylius/core": "^1.12",
         "sylius/currency": "^1.12",
         "sylius/order-bundle": "^1.12",
         "sylius/payment-bundle": "^1.12",
-        "sylius/resource-bundle": "^1.9"
+        "sylius/resource-bundle": "^1.9",
+        "php-http/httplug": "^2.4"
```

Comment:
Why is Httplug added as a requirement?

Reply:
I have no idea to be honest, there's a long weekend in Poland, so I'll try to catch Kevin once he becomes available.

Or maybe I remove the package and run the CI 😅.

Reply:
Hey! Httplug was added as Payum required it to work. I've not investigated it much, so I can't answer precisely why, but this because of the following exception:
```
[2023-04-28T07:49:21.500993+00:00] request.CRITICAL: Uncaught PHP Exception LogicException: "You cannot use "Symfony\Component\HttpClient\HttplugClient" as the "php-http/httplug" package is not installed. Try running "composer require php-http/httplug"." at /home/kevin/Projects/Sylius/Sylius/vendor/symfony/http-client/HttplugClient.php line 46
```
Payum seems to use HttplugClient in certain cases.

Reply:
This is caused by https://github.com/Payum/Payum/blob/master/src/Payum/Core/CoreGatewayFactory.php#L127.

I think it should be avoidable by configuring the respective Payum services explicitly, although I'm not 100% sure. 

Either way, it would be nice to also explicitly instruct Payum to use the Symfony Http client instead of letting it figure stuff out itself, if this is not already done. For same reasons as my initial report here.